src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,146 @@
+import { useContext, useEffect } from 'react';
+import { SearchLink } from './SearchLink';
+import { PeopleContext } from '../contsxts/PeopleContext';
+import { Person } from '../types';
+import cn from 'classnames';
+import { PeopleFilteredContext } from '../contsxts/PeopleFilteredContext';
```

Comment:
The import path for `PeopleFilteredContext` is incorrect. It should be '../contexts/PeopleFilteredContext' instead of '../contsxts/PeopleFilteredContext'.